// Copyright 2019 The MediaPipe Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package mediapipe;

import "mediapipe/framework/calculator.proto";

message CollectionHasMinSizeCalculatorOptions {
  extend CalculatorOptions {
    optional CollectionHasMinSizeCalculatorOptions ext = 259397840;
  }

  // The minimum size an input iterable collection should have for the
  // calculator to output true.
  optional int32 min_size = 1 [default = 0];
}
